manifests/service.pp

```diff
@@ -1,6 +1,7 @@
 class mcollective::service {
-  service{$mcollective::service_name:
+  service{ 'mcollective':
     ensure => $mcollective::service_ensure,
-    enable => $mcollective::service_enable
+    name   => $mcollective::service_name,
+    enable => $mcollective::service_enable,
```

Comment:
For this I prefer to stay with the prevailing style - not to put trailing `,`s.  It's only `missing` if it didnt work without it :P

Reply:
Okay, I'll remove this!